ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -364,21 +376,19 @@ private CompletableFuture<Block> processAnnouncedBlock(
     return getBlockFromPeers(Optional.of(peer), blockHash.number(), Optional.of(blockHash.hash()));
   }
 
-  private void requestParentBlock(final BlockHeader blockHeader) {
+  private void requestParentBlock(final Block block) {
+    BlockHeader blockHeader = block.getHeader();
     if (requestedBlocks.add(blockHeader.getParentHash())) {
       retrieveParentBlock(blockHeader);
     } else {
-      LOG.trace("Parent block with hash {} was already requested", blockHeader.getParentHash());
+      LOG.info("Parent block with hash {} is already requested", blockHeader.getParentHash());
```

Comment:
This seems like an overly noisy log message. debug seems reasonable, but this isn't something the user is likely to care about so shouldn't be at info level.

Reply:
Changed to debug